ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -1604,3 +1604,15 @@ jobs:
          template: *ci-master-latest
          timeout: 7200
          topology: *master_1repl_1client
+
+  fedora-latest/test_dns:
+     requires: [fedora-latest/build]
+     priority: 50
+     job:
+       class: RunPytest
+       args:
+         build_url: '{fedora-latest/build_url}'
+         test_suite: test_integration/test_dns.py
+         template: *ci-master-latest
+         timeout: 3600
```

Comment:
Hi @kaleemsiddiqu 
the timeout is not consistent in the various nightly definitions. Could you use 3600 everywhere?

Reply:
@flo-renaud 
I have updated 3600 across all files.